devicetypes/sensative/sensative-strips-drip-700.src/sensative-strips-drip-700.groovy

```diff
@@ -0,0 +1,617 @@
+/*
+ *  Sensative Strips Drip 700 v1.0
+ *
+ *
+ *  Changelog:
+ *
+ *    1.0 (05/12/2021)
+ *      - Initial Release
+ *
+ *
+ *  Copyright 2021 Sensative
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+*/
+
+import groovy.transform.Field
+
+@Field static Map commandClassVersions = [
+	0x22: 1,	// ApplicationStatus
+	0x31: 5,	// Sensor Multilevel (v7)
+	0x55: 1,	// Transport Service
+	0x59: 1,	// AssociationGrpInfo
+	0x5A: 1,	// DeviceResetLocally
+	0x5E: 2,	// ZwaveplusInfo
+	0x6C: 1,	// Supervision
+	0x70: 2,	// Configuration
+	0x71: 3,	// Alarm v1 or Notification v4
+	0x72: 2,	// ManufacturerSpecific
+	0x73: 1,	// Powerlevel
+	0x7A: 2,	// FirmwareUpdateMd
+	0x80: 1,	// Battery
+	0x84: 2,	// WakeUp
+	0x85: 2,	// Association
+	0x86: 1,	// Version (2)
+	0x87: 1,	// Indicator
+	0x8E: 2,	// Multi Channel Association
+	0x9F: 1 	// Security 2
+]
+
+@Field static int wakeUpIntervalSeconds = 43200
+@Field static int tempSensorType = 1 
+@Field static int waterSensorType = 31 
+@Field static int leakageCalibrationParamNum = 23
+
+metadata {
```

Comment:
Did You consider supporting Your both devices (this and the one from #66394) by one DTH ?
You would have to move ocfDeviceType, vid, mnmn from definition() to the fingerprints. Some tweaks in configuration code etc might be needed too (and some "if"-s) . 
That way You might avoid a code duplication.

Reply:
If SmartThings supported conditionally showing preferences based on which device was joined then using one handler to support both devices would be possible, but it doesn't and it also doesn't make sense to combine them because ones a Contact Sensor and the other is a Water/Temperature Sensor...

Reply:
Another reason these handlers can't be merged is that some of the built-in SmartApps don't use the the presentation so both devices would appear in their Water Sensor and Contact Sensor lists.

An updated version of this handler will be posted soon that includes the same types of changes you requested on PR #66394 so you might want to hold off on reviewing this until the new version is posted.